core-bundle/src/Resources/contao/templates/frontend/fe_page.html5

```diff
@@ -94,4 +94,4 @@
 
   <?= $this->mootools ?>
 </body>
-</html>
```

Comment:
this change should be reverted. You might want to add an `.editorconfig` plugin to your IDE 😉

Reply:
That's bundled with phpstorm, so 🤷